ipaserver/install/adtrust.py

```diff
@@ -451,6 +454,7 @@ class ADTrustInstallInterface(ServiceAdminInstallInterface):
         description="Add IPA masters to a list of hosts allowed to "
                     "serve information about users from trusted forests"
     )
+    add_agents = replica_install_only(add_agents)
```

Comment:
Is this fixing some kind of a regression? I don't mind if it's fixed in this commit, I would just like to know.